.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: otel/opentelemetry-go-agent:${{ steps.vars.outputs.tag }}
```

Comment:
It may be good to use https://github.com/docker/metadata-action

Example usage: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/.github/workflows/build_and_push_image.yml

Also you can see https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

Reply:
Where is the .Net image being pushed?  I don't see it in GitHub or DockerHub.

Reply:
That action is amazing, thanks for pointing it out.